### ISSUE TITLE
Hot reloading doesn't work in manage example app

### ISSUE THREAD

**Issue by nkohari** [auth0/cosmos#652]
Getting this message when starting up the manage example app:

```
ERROR in multi webpack-hot-client/client?40052f5f-6b3f-49f1-9420-b1606434fa51 ./index.js
Module not found: Error: Can't resolve 'webpack-hot-client/client?40052f5f-6b3f-49f1-9420-b1606434fa51' in '/Users/nkohari/Shared/Work/auth0/cosmos/examples/manage'
 @ multi webpack-hot-client/client?40052f5f-6b3f-49f1-9420-b1606434fa51 ./index.js
ℹ ｢wdm｣: Failed to compile.
```

The site still works, but it seems like hot reloading doesn't?

**Comment by siddharthkp** [comment]
Taken care in https://github.com/auth0/cosmos/pull/645

Can you test with that?

**Comment by siddharthkp** [comment]
Fixed in #645
